### PR TITLE
Use CSS Vars for Bubble Background and Text Color

### PR DESCRIPTION
**Pull request by peetzweg**
use css vars to easily change the bubble background and text color.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -12,10 +19,4 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  background-color: black;
-}
-
-code {
```
Comment (craftzdog) [inline src/index.css:18]: Please do not remove font family

**Review by craftzdog — CHANGES_REQUESTED** [review]

File: src/bubble.css [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+  display: inline-block;
   margin-bottom: 5px;
-}
-
-.bubble-content {
   margin-right: 25%;
-  background-color: #eee;
+  margin-top: 5px;
+  max-width: 600px;
+  padding: 10px 24px 10px 20px;
```
Comment (craftzdog) [inline src/bubble.css:14]: why not `padding: 12px 20px;`?

File: src/index.css [hunk 3]
```diff
@@ -12,10 +19,4 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  background-color: black;
```
Comment (craftzdog) [inline src/index.css:15]: I'd like the body bg to be kept black

File: src/index.css (hunk 1 above)
Comment (peetzweg) [reply src/index.css:18]: Afaik it's not used anywhere in the code base. Is there a feature I don't know about? Can't find any use of a `<code>` block in the source code. So I thought to just clean up a bit. :)

**Comment by peetzweg** [comment]
Added types throughout all files. ✌️ 

Was this jumping of bubbles an issue before as well? 🤔 Not to sure if I have broken it somewhere.

demo2

**Pull request merged**